lib/AST/ASTImporterLookupTable.cpp

```diff
@@ -56,6 +56,18 @@ struct Builder : RecursiveASTVisitor<Builder> {
     return true;
   }
 
+  // Functions without previous declaration are not visited by the AST visitor
+  // so they must be added explicitly (like the FriendDecl cases). These are
+  // referenced by the expression that calls them so find these.
+  bool VisitCallExpr(CallExpr *E) {
+    FunctionDecl *Callee = E->getDirectCallee();
+    // Our case: Declaration that is inside a function and implicit.
+    if (Callee && Callee->getLexicalDeclContext()->isFunctionOrMethod() &&
```

Comment:
Add a function `isCImplicit()` ?

Reply:
Probably not needed to add it to `FunctionDecl` because it is a rare case. And I am not sure if the conditions (for getLexicalDeclContext and isImplicit) are always correct and only true for this case.